src/components/Button/ButtonContent/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {
+	ButtonIcon,
+	ButtonTitle,
+	ButtonSubtitle,
+	ButtonCenter,
+	ButtonLine,
+} from './styled'
+
+export enum layoutType {
```

Comment:
First letter should be uppercase. :grin: 
```suggestion
export enum LayoutType {
```